man/makeTria.rd

```diff
@@ -0,0 +1,21 @@
+\name{makeTria}
+\alias{makeTria}
+\title{Reshapes an unordert covariance/correlation matrix into triangular shape}
+\description{Function is mainly used for \code{\link{wtdHetcor}} and
+\code{\link[eatModel]{q3FromRes}} functions in the eatModel package:
```

Comment:
Can a non-CRAN package be linked?

Reply:
I think this still needs to be resolved to be accepted at `CRAN`; packages can not be linked that are unknown to `CRAN`. The `\link{}` calls need to be removed IMHO.